examples/m5_gluonts_template.ipynb

```diff
@@ -0,0 +1,499 @@
+    "\n",
+    "By default the notebook is configured to run in submission mode (`submission` will be `True`), which means that we use all of the data for training and predict new values for a total length of `submission_prediction_length` for which we don't have ground truth values available (performance can be assessed by submitting prediction results to Kaggle). In contrast, setting `submission` to `False` will instead use the last `single_prediction_length`-many values of our training set as validation points (and hence these values will not be used for training), which enables us to validate our model's performance offline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "single_prediction_length = 28\n",
+    "submission_prediction_length = single_prediction_length * 2\n",
+    "m5_input_path=\"./m5-forecasting-accuracy\"\n",
+    "submission=True\n",
+    "\n",
+    "if submission:\n",
+    "    prediction_length = submission_prediction_length\n",
+    "else:\n",
+    "    prediction_length = single_prediction_length"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Reading the M5 data into GluonTS\n",
+    "\n",
+    "First we need to convert the provided M5 data into a format that is readable by GluonTS. At this point we assume that the M5 data, which can be downloaded from Kaggle, is present under `m5_input_path`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "calendar = pd.read_csv(f'{m5_input_path}/calendar.csv')\n",
+    "sales_train_validation = pd.read_csv(f'{m5_input_path}/sales_train_validation.csv')\n",
+    "sample_submission = pd.read_csv(f'{m5_input_path}/sample_submission.csv')\n",
+    "sell_prices = pd.read_csv(f'{m5_input_path}/sell_prices.csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We start the data convertion process by building dynamic features (features that change over time, just like the target values). Here, we are mainly interested in the event indicators `event_type_1` and `event_type_2`. We will mostly drop dynamic time features as GluonTS will automatically add some of these as part of many models' transformation chains."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cal_features = calendar.drop(['date', \n",
+    "                              'wm_yr_wk', \n",
+    "                              'weekday', \n",
+    "                              'wday', \n",
+    "                              'month', \n",
+    "                              'year', \n",
+    "                              'event_name_1', \n",
+    "                              'event_name_2', \n",
+    "                              'd'], axis=1)\n",
```

Comment:
Other minor comment: I think we use the following newline policy elsewhere in the codebase:

```
function(
    argument1, argument2, ...
)

[
    element1, element2, ...
]
```

(essentially, going to a newline right after parentheses, and not after the first element)

Reply:
Both comments fixed! :)